packages/web/src/components/ui/SidebarCategory/SidebarCategory.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import styles from './SidebarCategory.module.scss';
+import { Icon } from '../Icon/Icon';
+import clsx from 'clsx';
+import Badge from '../Badge/Badge';
+import Chip from '../Chip/Chip';
+import Person from '../Person/Person';
+import Checkbox from '../Checkbox/Checkbox';
+import SidebarCategorySearch from '../SidebarCategorySearch/SidebarCategorySearch';
+
+type listItem = {
+  id: string;
+  name: string;
+};
+
+type GroupItem = {
+  group: string;
+  children: listItem[];
+};
+
+type Group = {
+  [key: string]: GroupItem;
+};
+
+type Props = {
+  category?: {
+    fullList: listItem[];
+    featuredItems: string[];
+  };
+  simpleCategory?: string[];
+  selectedKeywords: string[] | [];
+  selectHandler: (name: string) => void;
+  renderComponent: 'chip' | 'checkbox' | 'person';
+  searchable?: boolean;
+};
+
+export default function SidebarCategory({
```

Comment:
The whole component looks messy, but I don't have any improvement suggestions right now. Left this comment to grab some attention here in future.